grpc/execution/validation.go

```diff
@@ -18,10 +18,16 @@ import (
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/holiman/uint256"
 )
 
+type AstriaTransaction struct {
```

Comment:
```suggestion
type AstriaTransactions struct {
```

since it contains multiple transactions? or `TransactionsWithType` to be more descriptive?

Reply:
Agreed! I went with `AstriaTransactionsWithType` and also updated some var names in server.go where this type is used to match.